cinema/serializers.py

```diff
@@ -68,13 +79,61 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
+
+    def validate(self, attrs):
+        Ticket.validate_seat_and_row(
```

Comment:
better call super() here and save the result, then after validation return result

Reply:
This to be 100% sure that you return the same thing that is in the parent class